pages/star-search/index.tsx

```diff
@@ -704,7 +705,26 @@ function Chatbox({ message, userId }: { message: StarSearchChat; userId?: number
     // Breaking all words so that the rendered markdown doesn't overflow the container
     // in certain cases where the content is a long string.
     content = (
-      <Markdown remarkPlugins={[remarkGfm]} className="break-words prose">
+      <Markdown
+        remarkPlugins={[remarkGfm]}
+        components={{
```

Comment:
The components prop allows you to manipulate any rendered components. In the case of markdown, this means standard HTML tags, so in our case, we render an avatar hovercard if the link in question is for a user. Otherwise the anchor, `<a>` tag renders normally.